runtests.py

```diff
@@ -476,13 +476,17 @@ def build_project(args):
 
     py_v_s = sysconfig.get_config_var('py_version_short')
     platlibdir = getattr(sys, 'platlibdir', '')  # Python3.9+
-    site_dir_template = sysconfig.get_path('platlib', expand=False)
+    site_dir_template = os.path.normpath(sysconfig.get_path(
+        'platlib', expand=False
+    ))
     site_dir = site_dir_template.format(platbase=dst_dir,
                                         py_version_short=py_v_s,
                                         platlibdir=platlibdir,
                                         base=dst_dir,
                                         )
-    noarch_template = sysconfig.get_path('purelib', expand=False)
+    noarch_template = os.path.normpath(sysconfig.get_path(
+        'purelib', expand=False
+    ))
```

Comment:
`sysconfig.get_path` always returns unix-style file-separators, which will result in mixed usage of `/` and `\` in windows paths (thus breaking one of the tests).
This change commit should definitely be backported.